src/utils/gaa-test.js

```diff
@@ -3722,8 +3750,44 @@ describes.realWin('GaaMetering', {}, () => {
       showPaywall = sandbox.fake();
       GaaMetering.userState = {};
 
-      sandbox.stub(GaaMetering, 'isUserRegistered');
-      GaaMetering.isUserRegistered.returns(true);
+      sandbox.stub(GaaMetering, 'isUserRegistered').returns(true);
+
+      const googleEntitlementsPromise = new Promise((resolve) => {
```

Comment:
This feels like an off pattern. Any reason something like 
```
Promise.resolve({ consume: () => {}, enablesThis: () => false, enablesThisWithGoogleMetering: () => false })
// or
Promise.resolve({ 
  consume: sandbox.fake,
  enablesThis: sandbox.fake.returns(false),
  enablesThisWithGoogleMetering: sandbox.fake.returns(false) 
});
```
wouldn't work?

Reply:
Good idea, thanks!